app/pages/Accounts/Transactions.module.scss

```diff
@@ -1,29 +1,7 @@
 @import 'config/bundle';
```

Comment:
`Transactions.module.scss` could probably be moved into the `TransactionList` folder, seeing as otherwise only `TransactionsAndAddress` use it, and only things from `_common`

Reply:
Wouldn't want to reference the `_common` file directly though. `TransactionsAndAddress` now references `Accounts.module.scss` instead.

Reply:
Yes, I was also thinking using `Accounts.module.scss` or creating it's own module.